static/styles/left-sidebar.css

```diff
@@ -190,6 +190,7 @@ ul.filters .arrow {
     position: absolute;
     right: 0px;
     top: 2px;
+    right: 4px;
```

Comment:
Moving other things by 4px would result in the `chevron` being too close to the `count` again. The reason why I made it 4px was because you mentioned it in the issue that having the `chevron` in the between the `scrollbar` and `count` would be better.